src/app/github-landing-page/github-landing-page.component.html

```diff
@@ -0,0 +1,95 @@
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+<div class="overflow-hidden">
+  <div class="mb-5 w-100 landing-page-container">
+    <div class="header-container w-100">
+      <h3>Please wait while we register your GitHub repository. This may take a few minutes.</h3>
+    </div>
+    <hr />
+    <h3 class="close-tab-message">You can close this tab and return to Dockstore on the original tab.</h3>
+
+    <div class="w-100 cards-container" fxLayout="row" fxLayoutAlign="space-evenly stretch">
+      <mat-card class="landing-page-card">
+        <mat-card-header>
+          <mat-card-title>Do you have a .dockstore.yml file in your repo?</mat-card-title>
+        </mat-card-header>
+        <mat-divider></mat-divider>
+        <mat-card-content class="h-100 landing-page-card-content dockstoreyml-repo-card-content">
+          <div class="landing-page-card-content-text">
+            If you haven’t already, please create a .dockstore.yml file in your repo. This way, your workflows will be kept up-to-date
+            automatically. <br />
+
+            Here is the .dockstore.yml template:
+          </div>
+
+          <pre>
+      version: 1.2
+      workflows:
+          - subclass: &lt;CWL | WDL | NFL | GALAXY&gt;
+          primaryDescriptorPath: &lt;String&gt;  # absolute path
+          testParameterFiles: &lt;String Array&gt;  # absolute path
+          readMePath: &lt;String&gt; # absolute path
+          authors:
+              - name: &lt;String&gt;
+                email: &lt;String&gt;
+          </pre>
+
+          <div class="landing-page-card-content-text">
+            Note: the first line of this file references the version of .dockstore.yml syntax being used, not the version/tag of the
+            workflow/tool/service/notebook it describes
+          </div>
+        </mat-card-content>
+      </mat-card>
+
+      <mat-card class="landing-page-card">
+        <mat-card-header>
+          <mat-card-title class="ml-2">Tips and Tricks</mat-card-title>
+        </mat-card-header>
+
+        <mat-divider></mat-divider>
+
+        <mat-card-content class="h-100 landing-page-card-content">
+          <ul>
+            <li>
+              Make sure your file is saved as <code>.dockstore.yml</code>, not <code>dockstore.yml</code> nor <code>.dockstore.yaml</code>
+            </li>
+            <li>Put the .dockstore.yml file in the top of your repo or inside the <code>.github/</code> directory</li>
+            <li>
+              <a href="https://qa.dockstore.org/quick-start">The Dockstore Yaml Command Line Validator Tool</a> can be used with the
```

Comment:
Shouldn't link to qa. I think you can make this a relative link, so you don't need the domain. Also you should use `routerLink`, so the whole page is not reloaded.